using-timescaledb/ingesting-data.md

```diff
@@ -0,0 +1,24 @@
+# Ingesting data
+
+TimescaleDB can support standard SQL inserts. Read more about how to use
+SQL to write data into TimescaleDB in our [Writing Data][writing-data] section.
+
+Users often choose to leverage existing 3rd party tools to build data ingest
+pipelines. Any tools that work with PostgreSQL also work with TimescaleDB.
```

Comment:
I think this might be a bit misleading cause there is a distinction here between reading and writing, in the monitoring ecosystem almost any tool can read stuff from postgres but only a small subset can actually write to postgres and for us this write support is actually needed to make it work with timescaledb.

Reply:
So I guess the fix here would be to update the Prometheus section with this language? I wasn't too sure how to flesh out this section since we already have a tutorial, so definitely grasping at straws.

Reply:
I think that repeating things in the docs is fine.  If there is a description of ingest that would apply in the tutorial, repeating that here is probably ok.

Reply:
Hmm... so eventually I was thinking that this page could just be a list of integrations that all work with Timescale to write data in, so this section in particular is light. I was thinking that we could just explain ingest vs read in the Prometheus section, and leave this section as it is right now?

Reply:
Ok cool - I just re-wrote it to talk more explicitly about both reading and writing using the adapter etc.